### PR TITLE
Update aneesh-gopalakrishnan.json

### PR DESCRIPTION
**Pull request by codehippie1**
Feed urls updated

# #WeAreDonet Creator Submission Guidelines

* [x] Check all points before submitting the PR.

* [x] By submitting this pull request to add my creator card, I verify that I adhere to the #WeAreDotnet creator card submission guidelines.

* [x] Make sure to follow our [Code of Conduct](https://github.com/wearedotnet/dotnet-creator-cards/blob/master/CODE_OF_CONDUCT.md) 

* [x] We do not accept submissions with the sole intention of promoting a product or company.

* [x]  We add you as a creator, not your company. 

* [x]  The moderator team decides whether we accept or reject a submission.

## When Do We Accept Someone's Creator Card Submission? 

* [x] You must have published at least three post (long form content) since 2022 on any of your channels. 

* [x] There must be three content pieces in English. 

* [x] There needs to be more than tweets. 

* [x] Longer-form article on LinkedIn is good, but a self-hosted blog or one on a well-known platform like Hashnode, medium etc., is better. 

* [x] Alternatively, you contributed with another form on a platform like YouTube, Twitch, Udemy or Pluralsight. If you have some recorded talks of conferences or anywhere else, that counts too! So in the form of a recorded video or a book. 

## RSS or Atom Feed 

* [x] Submit an Atom or RSS URL inside your creator card that belongs to your blog 

* [x] If you do not submit an RSS or Atom URL. We cannot consider your feed for the upcoming aggregation feature. 

## Content Guideline 

* [x] If you have a self-hosted blog, please ensure you have a valid HTTPS certificate. 

* [x] We host NO malicious or offensive content on the blog (including photos, swearing, etc.)

* [x] You must add the full URL to your GitHub profile to your card, so we can assign it to the member account that used the same GitHub account for sign-up.

### CODE REVIEW

**Comment by matthiasjost** [comment]
@codehippie1 It looks like we have two times the same URL now. @DeeJayTC temporarily removed you from the landing page but will enable you again as soon as we know everything is going well.

File: creator-cards/aneesh-gopalakrishnan.json [hunk 1]
```diff
@@ -19,8 +19,12 @@
   ],
   "feeds": [
     {
-      "type": "Blog",
-      "url": "https://rss.app/feeds/79Dl0OyyexNBxDuA.xml"
-    }
+      "type": "LinkTagRssType",
+      "url": "https://github.com/codehippie1/feeds/blob/main/aneesh_blogs.xml?raw=true"
```
Comment (DeeJayTC) [inline creator-cards/aneesh-gopalakrishnan.json:23]: i get a 404 trying to open this?

**Comment by codehippie1** [comment]
Sorry about that. I created a simple github page to host the feed XML files. Let me know if the updated changes work fine.

File: creator-cards/aneesh-gopalakrishnan.json [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
     }
   ],
-  "feeds": [
+  "projects": [
     {
-      "type": "Blog",
-      "url": "https://rss.app/feeds/79Dl0OyyexNBxDuA.xml"
+      "name": "ng2-pdfjs-viewer",
+      "description": "Angular PDF Viewer",
+      "url": "https://github.com/intbot/ng2-pdfjs-viewer"
```
Comment (DeeJayTC) [inline creator-cards/aneesh-gopalakrishnan.json:27]: Can you tell me why this is related to .NET? Does not look like a .NET Project to me.

**Comment by codehippie1** [comment]
@DeeJayTC - You are right. It is not a .NET Project. I can remove it if you see fit.

Backstory.
It was created to fill the gap in aspnet core when it removed support for RDLC Report viewer.
Link - https://learn.microsoft.com/en-us/archive/blogs/sqlrsteamblog/microsoft-acquires-report-rendering-technology-from-forerunner-software

The original thread which eventually gave way to the project - Link: https://stackoverflow.com/questions/49309797/rdlc-local-report-viewer-for-asp-net-core-and-angular2-0

The blog post has a section for asp.net usage under "A sample API using C# and ASP.NET Core"
Also, upon analysis, I have a good clientele using aspnet core with angular.

**Comment by DeeJayTC** [comment]
We do allow angular content when its in a blog, video etc something like building an app with .NET and Angular..totally fine but the project section will be solely for .NET Projects of any kind.

**Comment by codehippie1** [comment]
@DeeJayTC Makes sense. Removed it from "projects" section.

**Comment by DeeJayTC** [comment]
Thats fine now, ill have you updated by tomorrow.

**Pull request merged**